library/blivet.py

```diff
@@ -220,7 +224,7 @@ def _new_mdarray(self, members, raid_name=""):
                     requested_actives < 0 or requested_spares < 0):
                 raise BlivetAnsibleError("failed to set up volume '%s': cannot create RAID "
                                         "with %s members (%s active and %s spare)"
-                                        % (self._volume["name"], len(members),
+                                        % (self._volume["name"], len(members),  # pylint: disable=no-member
```

Comment:
Is this method not used for pools? If so, it must use `self._spec_dict` instead of either `self._volume` or `self._pool`; if not, it should be in the `BlivetVolume` class instead.

Reply:
Fixed. Replaced with `self._spec_dict`. Removed `pylint: disable`.